pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -26,8 +29,16 @@ buildPythonPackage rec {
 
   doCheck = false;
 
-  nativeBuildInputs = [ pkgconfig which doxygen wxGTK ];
-  buildInputs = [ ncurses wxGTK.gtk ];
+  nativeBuildInputs = [ pkgconfig which doxygen ]
```

Comment:
```suggestion
  nativeBuildInputs = [ pkg-config which doxygen ]
```

Reply:
I'll have to check that, did that change recently?

Reply:
We renamed pkgconfig to pkg-config but if this is the python module that it is still the same name. I am confused now.

Reply:
Everything in `pkgs/top-level/python-packages.nix` passes `pkgs.pkgconfig`. Doesn't look to me that we should change that here.

Reply:
There is a python package named pkgconfig and it would be great to distinguish them. Also eventually we want to remove the alias to pkgconfig. Also when things got changed recently we can't rely on how it is done everywhere else if everywhere else might not be changed yet.